src/app/shared/components/footer/footer.component.html

```diff
@@ -59,6 +59,14 @@
           all content is licensed under' | translate}}{{'a creative commons Attibution-No Derivates 4.0 International License' | translate}}
         </div>
       </div>
+      <div class="stripo-logo-container">
+        <p class="title-text">Newsletter Powered by:</p>
+        <picture>
+          <source srcset="assets/images/stripo-email.webp" type="image/webp">
+          <source srcset="assets/images/stripo-email.jpg" type="image/jpeg">
+          <img src="assets/images/stripo-email.jpg" alt="stripo email logo" />
```

Comment:
Not sure what's being done here as the uploaded image format is `png` but the code is referring to a `jpg` image

Reply:
Fixed.